inference-engine/thirdparty/clDNN/kernel_selector/core/common/jitter.cpp

```diff
@@ -1833,11 +1835,8 @@ std::string FusedOpsCodeGenerator::Broadcast(std::string var, Datatype dt, size_
     return "(" + GetType(dt, vec_size) + ")(" + var + ")";
 }
 
-std::string FusedOpsCodeGenerator::ConvertToOutputTypeSat(std::string var, size_t vec_size) const {
-    if (desc.output_tensor.GetDType() == Datatype::F32 || desc.output_tensor.GetDType() == Datatype::F16)
-        return "convert_" + GetOutputType(vec_size) + "(" + var + ")";
-    else
-        return "convert_" + GetOutputType(vec_size) + "_sat(" + var + ")";
+std::string FusedOpsCodeGenerator::ConvertToOutputTypeRte(std::string var, size_t vec_size) const {
+    return "convert_" + GetOutputType(vec_size) + "_rte(" + var + ")";
```

Comment:
I think convert_*_sat_rte should be used here.

Reply:
Thanks, I'll check it.